DllImportGenerator/DllImportGenerator/Resources.resx

```diff
@@ -181,6 +181,15 @@
   <data name="CustomTypeMarshallingNativeToManagedUnsupported" xml:space="preserve">
     <value>The specified parameter needs to be marshalled from native to managed, but the native type '{0}' does not support it.</value>
   </data>
+  <data name="GeneratedDllImportContainingTypeMissingModifiersDescription" xml:space="preserve">
+    <value>Types that contain methods marked with 'GeneratedDllImportAttribute' must be 'partial'. P/Invoke source generation will ignore methods contained within non-partial types.</value>
```

Comment:
>will ignore methods contained within non-partial types

Why ignore as opposed to erroring out?

Reply:
Because that's the behavior we have today for the other case where we see a non-partial method with this attribute.

Reply:
Okay. I thought we were erroring out if we found a case where the attribute was used in a way we couldn't generate.